style.css

```diff
@@ -611,3 +620,808 @@ body {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #fff;
+    margin-left: 120px;
+    margin-top: 10px;
+  }
+
+  #resume-btn button {
+    padding: 25px;
+    gap: 10px;
+    width: 161px;
+    height: 48px;
+    background: #ff6b00;
+    font-family: 'Inter', sans-serif, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 24px;
+    color: #fff;
+    margin-left: 120px;
+    margin-top: 30px;
+  }
+
+  #resume-btn button:hover {
+    color: #f55800;
+    background-color: #fff5e1;
+    border: 1px solid #f55800;
+  }
+
+  #resume-btn button:active {
+    color: #fff;
+    border: none;
+    background-color: #ff6b00;
+    width: 171px;
+    height: 58px;
+    left: 407px;
+    top: calc(50% - 48px / 2 + 46.5px);
+  }
+
+  #resume-btn button:disabled {
+    width: 161px;
+    height: 48px;
+    left: 564px;
+    top: calc(50% - 48px / 2 + 46.5px);
+    background: #f4f1f0;
+  }
+
+  #about-bg {
+    display: block;
+    position: absolute;
+    top: 12%;
+    left: 61.4%;
+  }
+
+  #frame-bg {
+    display: none;
+  }
+
+  #line-2 {
+    width: 1250px;
+    position: absolute;
+    top: 32%;
+    left: 9.1%;
+  }
+
+  .language-header {
+    display: flex;
+    flex-direction: row;
+    width: 1157px;
+    height: 596px;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 200px;
+    margin-left: 125px;
+    margin-top: -40px;
+  }
+
+  .header-6 {
+    font-family: 'InaiMathi', 'Inter', sans-serif, 'Times New Roman', Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    list-style: none;
+    margin-top: 0;
+  }
+
+  .list {
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+    margin-top: 40px;
+    margin-left: -10px;
+    padding: 0;
+  }
+
+  #skills-list {
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+    margin-top: 40px;
+    margin-left: 0;
+    padding: 0;
+  }
+
+  #Languages-list {
+    margin-top: -70px;
+  }
+
+  .form-section {
+    width: 1520px;
+    height: inherit;
+  }
+
+  .form-content {
+    width: 1463px;
+    height: 590px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: -40px;
+  }
+
+  .form-heading {
+    display: none;
+  }
+
+  .form-heading-1 {
+    width: 455px;
+    height: 256px;
+    display: flex;
+    margin-bottom: 170px;
+  }
+
+  .header-7 {
+    font-family: "Inter", sans-serif, 'Times New Roman', Times, serif;
+    font-weight: 700;
+    font-size: 40px;
+    font-style: normal;
+    line-height: 52px;
+    text-align: start;
+    color: #262626;
+    margin: 0;
+  }
+
+  .form-container {
+    width: 700px;
+    height: 630px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 200px;
+    margin-left: 40px;
+  }
```

Comment:
- [ Optional ] The Minimum mobile screen size should be 375px , so it makes responsive your design I recommend you to adjust the alignment of the Forms for contact section just like the Figma design . 

Your design for contact section : Form container look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/193092725-3ecabcc3-a0f8-4c38-922c-4e8d5ef75650.PNG)

The Figma design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/193092408-56199280-fb00-4e7c-ac17-edb070cb6a47.PNG)